src/applications/disability-benefits/all-claims/tests/all-claims-add-disabilities.cypress.spec.js

```diff
@@ -27,10 +19,10 @@ const testConfig = createTestConfig(
       data: path.join(__dirname, 'fixtures', 'data'),
     },
 
-    pageHooks: pageHooks(cy, { toggles }),
+    pageHooks: pageHooks(cy),
     setupPerTest: () => {
       cy.login(mockUser);
-      setup(cy, { toggles });
```

Comment:
hmm, i think the reason this new spec was added was so that we could test both the old and revised page. note how dataSet `new-only-test` (line 16) is also configured to run with src/applications/disability-benefits/all-claims/tests/all-claims.cypress.spec.js. I think we can delete this spec